index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="index.html" method="post">
```

Comment:
- As requested by the previous reviewer, kindly follow the project requirement and implement the interaction in your form so it gets submitted using the [formspree service](https://formspree.io/html)

Reply:
I did, just look at the second commit entitled [change on form, improving action attribute](https://github.com/GhislainMitahi/M_portfolio/pull/5/commits/ba74c2064f70d71fd5ccbecace1a99d424409575)